introduction.rst

```diff
@@ -105,12 +106,43 @@ Conan is being used in production by thousands of companies like TomTom, Audi, R
 
 But an essential part of Conan is that many of those users will contribute back, creating an amazing and helpful community:
 
-- The https://github.com/conan-io/conan project has around 6K stars in Github and counts with contributions of almost 300 different users (this is just the client tool).
+- The https://github.com/conan-io/conan project has around 6.5K stars in Github and counts with contributions from more than 300 different users (this is just the client tool).
 - Many other users contribute recipes for ConanCenter via the https://github.com/conan-io/conan-center-index repo, creating packages for popular Open Source libraries, contributing many thousands of Pull Requests per year.
 - More than two thousands Conan users hang around the `CppLang Slack #conan channel <https://cppalliance.org/slack/>`_, and help responding to questions, discussing problems and approaches, making it one of the most active channels in the whole CppLang slack.
 - There is a Conan channel in `#include<cpp> discord <https://www.includecpp.org/discord/>`_.
 
 
+Navigating the documentation
+------------------------------
+
+This documentation has very different sections:
+
+- The **tutorial** is an actual hands-on tutorial, with examples and real code, intended to be played sequentially from beginning to end,
+  running the exercises in your own computer. There is a conducting thread and exercises might depend on some previous explanations and code.
+  This is the recommended approach for learning Conan.
+- The **examples** also contains hands-on, fully operational examples with code, aimed to explain some very specific feature, tool or behavior.
+  They do not have a conducting thread, they should be navigated by topic.
+- The **reference** is the source of truth for the interfaces of every public command, class, method, helper, API and configuration file that can
+  be used. It is not designed to be read fully, but to check for individual items when necessary.
+- The **knowledge** base contains things like the FAQ, a very important section about genral guidelines, good practices and bad practices, videos
+  from conference talks, etc.
+
+Features in this documentation might be labeled as:
+
+- **experimental**: This feature is released and can be used, but it is under active development and the interfaces, APIs or behavior might change
+  as a result of evolution, and this will not be considered breaking. If you are interested in these features you are encouraged to try them
+  and give feedback, because that is exactly what allows to stabilize them.
+- **preview**: When a feature is released in preview mode, this means it aims to be as final and stable as possible. Users are encouraged to use them,
```

Comment:
`final and stable as possible` doesn't play well together with `if necessary, they might change and break`. second part basically means `unstable and subject to change`. it's either `stable` or `experiment`, and practically, right now definition of `preview` is not actually different from `experimental` (maybe, just with probability of changes, like `preview` has 1% chance to change in a breaking way, while `experimental` has 99% change to break, but both may break your code, ultimately).